### ISSUE TITLE
Investigate content negotiation in Jekyll

### ISSUE THREAD

**Issue by tidoust** [w3c/web-roadmaps#248]
See context in https://github.com/w3c/web-roadmaps/issues/68#issuecomment-338312178.

It would be good to enable content negotiation for roadmaps published on GitHub Pages so that e.g. users get the Chinese version when Chinese is their primary language. GitHub Pages use [Jekyll](https://jekyllrb.com/) under the hoods, so goal is to check whether that can be done in Jekyll.

This may require switching to different naming conventions for HTML files, which we may or may not want to do.

**Comment by tidoust** [comment]
Not sure what I had in mind when I created that issue but GitHub Pages is a [static page hosting service](https://help.github.com/articles/what-is-github-pages/) and [does not support customer server configuration files](https://help.github.com/articles/redirects-on-github-pages/), so the only thing that we can hope to achieve for content negotiation would be at the HTML level, e.g. through an `<meta http-equiv="refresh">` tag, but more likely through JS (since we want to redirect based on the browser's locale). Jekyll won't help us for that.

In other words, the solution for GitHub Pages would be to have an "empty" generic page that redirects to the right version using JS. I'm not really crazy about introducing that first empty page and about requiring support for JS (we could fallback to redirecting to the English version with a meta tag after a couple of seconds though).
